third-party/patches/libvterm-arm64-darwin.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/bin/vterm-ctrl.c b/bin/vterm-ctrl.c
+index ba0d61e..8f8be7e 100644
+--- a/bin/vterm-ctrl.c
++++ b/bin/vterm-ctrl.c
+@@ -1,4 +1,4 @@
+-#define _XOPEN_SOURCE 500  /* strdup */
++#define _XOPEN_SOURCE 600  /* strdup */
```

Comment:
This should be fixed upstream.  Cc @leonerd

Reply:
Reverted the commit and created neovim/libvterm#6.

Reply:
Updated the PR to use the newest revision of libvterm and adapted a line of code.

Reply:
> and adapted a line of code.

That's an unreleased API change.  We shouldn't be pulling that in.  @leonerd, any chance of a maintenance release while the mainline stabilizes?

Reply:
Ups, reverted the commit.